README.md

```diff
@@ -108,6 +108,7 @@ eza’s options are almost, but not quite, entirely unlike `ls`’s.
 - **-D**, **--only-dirs**: list only directories
 - **-f**, **--only-files**: list only files
 - **--git-ignore**: ignore files mentioned in `.gitignore`
+- **--ignore-submodule-contents**: do not list contents of submodules
```

Comment:
For the flag name, how about `--git-ignore-submodules` ? Either way it's going to be a long flag but I'd like to get others thoughts on this. @eza-community/eza-commiters

Reply:
Hm, it is quite long ... I think `--ignore-submodules` or `--ignore-submods` would suffice 🤔